conda/recipes/versions.yaml

```diff
@@ -13,9 +13,9 @@ xgboost_version:
 
 # Versions for conda
 conda_version:
-  - '=4.10.3'
+  - '=4.12.0'
 conda_build_version:
-  - '=3.21.7'
+  - '=3.21.8'
 conda_verify_version:
   - '=3.1.1'
```

Comment:
We may want to update `conda-verify` later, but given what it is pinned to it might be worth handling that separately